libpod/pod.go

```diff
@@ -59,6 +59,20 @@ type podState struct {
 	CgroupPath string
 }
 
+// PodInspect represents the data we want to display for
+// podman pod inspect
+type PodInspect struct {
```

Comment:
These should probably go in pkg/inspect/inspect.go with the other structs from inspect

Reply:
I can live without moving it, just wanted to do so if it was easy